src/components/Layout/Footer/Footer.module.scss

```diff
@@ -0,0 +1,5 @@
+.Footer {
+  background-color: bisque;
```

Comment:
can you make this a variable?

Reply:
It's only that colour as a temporary measure